packages/next-sanity/src/image/imageLoader.ts

```diff
@@ -6,7 +6,7 @@ import type {ImageLoader} from 'next/image'
 export const imageLoader = (({src, width, quality}) => {
   const url = new URL(src)
   url.searchParams.set('auto', 'format')
-  url.searchParams.set('fit', url.searchParams.get('fit') || 'min')
+  url.searchParams.set('fit', url.searchParams.get('fit') || 'max')
```

Comment:
I'm willing to make this change if it's applied when height isn't set, so that we don't break existing apps that rely on the image being clipped when both width and height is set:
```suggestion
  url.searchParams.set('fit', url.searchParams.get('fit') || url.searchParams.has('h') ? 
 'min' : 'max')
```